.Lib9c.Tests/Action/RankingBattleTest.cs

```diff
@@ -0,0 +1,99 @@
+namespace Lib9c.Tests.Action
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Collections.Immutable;
+    using System.Linq;
+    using Bencodex.Types;
+    using Libplanet;
+    using Libplanet.Crypto;
+    using Nekoyume;
+    using Nekoyume.Action;
+    using Nekoyume.Model.State;
+    using Nekoyume.TableData;
+    using Xunit;
+
+    public class RankingBattleTest : IDisposable
+    {
+        private TableSheetsState _tableSheetsState;
+
+        public RankingBattleTest()
+        {
+            _tableSheetsState = TableSheetsImporter.ImportTableSheets();
+        }
+
+        public void Dispose()
```

Comment:
I guess it's okay that we skip this `Dispose()`. (`null` assigning doesn't affect other test cases.)